src/base/bittorrent/torrentimpl.cpp

```diff
@@ -1153,30 +1153,18 @@ int TorrentImpl::leechsCount() const
 
 int TorrentImpl::totalSeedsCount() const
 {
-    return (m_nativeStatus.num_complete > 0) ? m_nativeStatus.num_complete : m_nativeStatus.list_seeds;
```

Comment:
IIRC there was a reason for using `0` but I can't recall it, or it could just be an coding error... Anyway just keep an eye on new bug reports.